tpu-client-next/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "solana-tpu-client-next"
+description = "Client code to send transaction to TPU."
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+async-trait = { workspace = true }
+log = { workspace = true }
```

Comment:
btw you can just write e.g.:
```rust
log.workspace = true
```
;)

Reply:
I do this way because in other crate we stick with this style